website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,64 @@
+
+Activating Karpenter on your cluster is a combination of a helm chart, and configuring the cloud platform to accept provisioning requests from Karpenter. On AWS, IAM roles for Service Accounts (IRSA) is used. The Kubernetes control plane in EKS oversees cluster-space Karpenter requests being securely elevated and passed on to the cloud platform. 
+
+## Provisioner CRD
+
+[Provisioner]({{< ref "/docs/provisioner-crd.md" >}}) is the primary Custom Resource Definition (CRD) for Karpenter, and you need at least one. 
+
+The primary kubernetes API object for Karpenter is “provisioner”. Notably, one provisioner can handle multiple node profiles (graphics enabled, compute optimized, memory optimized, etc). Karpenter is group-less, and eliminates management of multiple node groups with fixed instance specs. In short, the Karpenter provisioner object is focused on podspecs. This simplifies cluster management, and reduces the complexity of implementing Karpenter. 
+
+## Provsioning Procedure
+
+Karpenter activates when a pod is unschedulable. In response, it starts provisioning a new node, including the cloud provider instance. 
+
+Karpenter makes new instances based on the Provisioner CRD. In the default configuration, Karpenter automatically selects from all the general purpouse instances available, with an intent to accomodate the unscheduled pods efficently. Alternatively, the Provisioner CRD can define a list of acceptable instance types. 
+
+The Provisioner CRD includes lables for the new nodes. Certain labels, such as `kubernetes.io/arch`, change the instances Karpetner will provision. For example, setting `kubernetes.io/arch=arm64` configures Karpenter to provision ARM based instances. 
+
+[[todo: right float image]]
+[[todo: land label docs in main, then add links]]
+
+{{< imgproc fig1 Fit "400x450" >}}
+{{< /imgproc >}}
+
+### Binpacking
+
+{{< imgproc fig2 Fit "400x450" >}}
+{{< /imgproc >}}
+
+## Reallocation
+
+Currently, Karpenter does not support reallocation. A large instance with few pods will not be terminated based on usage. Review the deprovisioning triggers below. 
+
+## Deprovisioning Triggers
+
+Currently, Karpenter supports deprovisioning instances based on two metrics:
+- number of seconds from node creation, known as "node expiry"
+- number of seconds with no pods
+
+### Node Expiry
+
+Setting a value for `ttlSecondsUntilExpired` enables node expiration. The value is the number of seconds after node creation until nodes are viewed as expired by Karpenter. Note, with this value set, all nodes will eventually expire. Expired nodes are drained and replaced. The replacement nodes will have the latest updates, and may be more efficiently sized.
+
+### Empty Nodes
+
+Setting a value for `ttlSecondsAfterEmpty` enables deprovisioning empty nodes (no pods besides daemon sets). This only happens if a node becomes empty, and stays empty for the set number of seconds. 
```

Comment:
@njtran can help clarify all of the cases that are considered beyond daemonset pods.